src/blocks/stars.scss

```diff
@@ -0,0 +1,30 @@
+.product__rating {
+  display: flex;
+  align-items: center;
+}
```

Comment:
it should be separate

Reply:
what do you mean here?

Reply:
it should be in the product block not here

Reply:
thank you, it was duplicated. fixed